### PR TITLE
feat: Adds strike-through animation for the completed tasks

### LINKED ISSUE #20: Strike animation
When completing a text,

Instead of a simple `line-through` effect on top of the text,

I would like to be animated from left to right.

### PR DESCRIPTION
**Pull request by patelprabhuteja**
Adds strike-through animation for github issue #20

### CODE REVIEW

**Comment by patelprabhuteja** [comment]
https://github.com/liminalminds/karo/assets/64630166/ab7352a0-91fe-49b6-a370-2cbd9f0a0ebb

Attaching working demo for reference.

File: frontend/app/global.css [hunk 1]
```diff
@@ -193,9 +193,27 @@ main .tasks{
...
 .tasks .done{
-	text-decoration: line-through;
+	position: relative;
 	color: var(--lighter-grey);
+
+	&::before {
+		content: '';
+		position: absolute;
+		top: 50%;
+		height: 2px;
```
Comment (sunnybeta) [inline frontend/app/global.css:213]: Can you please change it to `4px`.

File: frontend/app/global.css [hunk 2]
```diff
@@ -193,9 +193,27 @@ main .tasks{
...
+		background-color: var(--lighter-grey);
+		animation: strike-through 2s forwards;
```
Comment (sunnybeta) [inline frontend/app/global.css:215]: Keep the time down to `0.5s` and add `linear` time

Comment (sunnybeta) [inline frontend/app/global.css:215]: Please keep the time down to `0.5s` and add `linear` time.

**Review by sunnybeta — DISMISSED** [review]
Require updating time and speed of the strike-through.

**Pull request merged**